package.json

```diff
@@ -40,14 +40,17 @@
     "rebuild-ckeditor": "cd ./public/lesswrong-editor && yarn install && yarn run build",
     "production": "scripts/runProduction.sh",
     "deploy": "scripts/deploy.sh",
-    "ea-command": "./build.js -run --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/dev-pg-conn.txt --command",
+    "ea-command-dev": "./build.js -run --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/dev-pg-conn.txt --command",
+    "ea-command-staging-db": "./build.js -run --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/dev-pg-conn.txt --command",
+    "ea-command-prod-db": "./build.js -run --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/dev-pg-conn.txt --command",
+    "ea-command-testing": "./build.js -run --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/testing-pg-conn.txt --command",
```

Comment:
I think it would be better if there were still just one `ea-command` script, but you have to pass in "testing" or "dev etc as a parameter, just to avoid bloating this file. Fine to merge as is though

Reply:
We'd have to edit `build.js` for that, which seems overkill for this niche use-case